src/pages/ApiInfo/ApiInfo.tsx

```diff
@@ -102,7 +105,17 @@ export const ApiInfo: React.FC = () => {
     <Drawer className={styles.apiInfo} size="medium" position="end" open onOpenChange={handleClose}>
       <DrawerHeader>
         <DrawerHeaderTitle
-          action={<Button appearance="subtle" aria-label="Close" icon={<Dismiss24Regular />} onClick={handleClose} />}
+          action={
+            <Button
+              appearance="subtle"
+              aria-label="Close"
+              icon={<Dismiss24Regular />}
+              onClick={() => {
+                document.title = 'API Center portal'; // Reset title to default
+                handleClose();
+              }}
```

Comment:
1. All callbacks must be memoized with useCallback (otherwise your component will re-render when unnecessary) (https://react.dev/reference/react/useCallback#skipping-re-rendering-of-components)
2. Looks like this title reset should happen in Home page and not here otherwise it knows details about its parent and we lose an ability to reuse it. As far as I can see title is not reset automatically in Home because Home is not unmounted so `useEffect` in `useDocumentTitle` for home is not triggered. I see only one way to fix it without an overkill: replace `useDocumentTitle` hook with a simple function and in Home page do this:
```
const nestedRoute = useOutlet();
if (!nestedRoute) {
  setPageTitle('API Center portal');
}
```